### DISCUSSION TITLE
No docker image for 4.0.5

### DISCUSSION THREAD

**Discussion by Stono** [rabbitmq/rabbitmq-server#12975] [Questions] [answered]
### Describe the bug

Hey, 
Sorry not sure if this is a "bug" (in the build process?) or more of a discussion topic?  So apologies if its in the wrong place.

I noticed 4.0.5 is out (great!) but there's still no docker image in https://hub.docker.com/_/rabbitmq for it?  Not sure the usual process for this project, is there a latency between the two or has something gone wrong?

ta

### Reproduction steps

See https://hub.docker.com/_/rabbitmq

### Expected behavior

The docker image to be available as a release artifact

**Comment by lukebakken** [comment, marked as answer]
Thanks for using RabbitMQ.

* Team RabbitMQ _does not_ maintain the community docker images.
* Be patient. It takes time to build docker images.
* In the future, start discussions here: https://github.com/docker-library/rabbitmq/discussions
